slave/slave_files/files/home/jenkins-slave/cleanup_docker_images.py

```diff
@@ -40,10 +41,9 @@ def get_image_list():
     images.extend(dangling_images)
     return reversed(images)
 
-
 def remove_docker_image(imageid):
     cmd = ("docker rmi %s" % imageid).split()
-    subprocess.check_call(cmd)
+    subprocess.check_output(cmd, stderr=subprocess.STDOUT)
```

Comment:
It ignores any output if the command succeeds. It gets captured in the exception in case of failure.